libs/txnbuild/price_update.go

```diff
@@ -50,6 +53,8 @@ func (priceUpdate *PriceUpdate) FromXDR(vXdrOp vxdr.VeloOp) error {
 	return nil
 }
 
+// Validate for PriceUpdate validates the required struct fields. It returns an error if any of the fields are
```

Comment:
Validation function for PriceUpdate. Validates the required struct fields. It returns an error if any of the fields are